core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -25,6 +25,11 @@
 @Table(name = "USERS")
 public class User extends EntityWithID
 {
+	public enum Status
```

Comment:
Suppose the user quits the app before the confirmation. Then a user has been created but not confirmed. How will the app know that the user should still be confirmed when opening the app the second time? There is no GET request to get the status, and no error will be thrown.

(Resending the confirmation SMS is also not implemented yet. I think this may be necessary because SMSes may be lost on the way. But if we can drop the user and recreate it that is maybe sufficient for version 1.)